docs/package/gluon-ssid-changer.md

```diff
@@ -0,0 +1,70 @@
+gluon-ssid-changer
+==================
+
+This package adds a script to change the SSID to an Offline-SSID when there is
+no connection to any gateway.
+This SSID can be generated from the nodes hostname with the first
+and last part of the nodename or the mac address, to allow observers to 
+recognise which node is down. The script is called once a minute by micron.d
+and it will change from online to offline-SSID maximum once every (definable)
+timeframe.
+
+You can enable/disable it in the config mode.
+
+Once every timeframe it checks if there's still a gateway reachable. There can be
```

Comment:
It checks if a gateway is reachable once in an interval. Different algorithms can be selected to determine whether a gateway is reachable.